### PR TITLE
chore: merge develop into 10.0-release

### PR DESCRIPTION
**Pull request by davidmunechika**
Merge develop into 10.0-release.

No squash merge. Do a normal merge commit.

### CODE REVIEW

File: packages/driver/cypress/e2e/cypress/command_spec.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+const { Command } = Cypress
+
+describe('driver/src/cypress/command', () => {
```
Comment (emilyrohrbough) [inline packages/driver/cypress/e2e/cypress/command_spec.js:3]: This should be renamed to `command.cy.js`

File: packages/config/lib/options.ts [hunk 2]
```diff
@@ -0,0 +1,524 @@
+const validate = require('./validation')
```
Comment (emilyrohrbough) [inline packages/config/lib/options.ts:1]: This file shouldn't be added. in 10.0-release branch, this content is under src/options.ts. Should be a small update to that file.

**Comment by lmiller1990** [comment]
Seems [cy.pause() in run mode does not pause without --no-exit [firefox]](https://app.circleci.com/pipelines/github/cypress-io/cypress/36117/workflows/2acacd47-9b0a-4675-a8a6-93b5e3b1c3f3/jobs/1443800/tests#failed-test-0) is still failing :thinking:

**Review by flotwig — APPROVED** [review]

**Comment by lmiller1990** [comment]
@davidmunechika I think we probably want to create a ticket to follow-up on why this is failing - if we skip this now, it's unlikely anyone will realize or remember to fix it (ever). Could you do that? Thanks! Probably include a link to this PR for reference.

**Comment by davidmunechika** [comment]
Ticket to address failing test: https://cypress-io.atlassian.net/browse/UNIFY-1567?atlOrigin=eyJpIjoiZTEzMTI3ZDA4YzA3NGVhM2E0YmVjZDE5ODkyY2ViMjgiLCJwIjoiamlyYS1zbGFjay1pbnQifQ

**Review by lmiller1990 — APPROVED** [review]

**Pull request merged**